docs/language/learn-ql/cpp/value-numbering-hash-cons.rst

```diff
@@ -55,9 +59,10 @@ To get the ``GVN`` of an ``Expr``, use the ``globalValueNumber`` predicate.
 
 Why not a predicate?
 ~~~~~~~~~~~~~~~~~~~~
+
 The obvious interface for this library would be a predicate ``equivalent(Expr e1, Expr e2)``. However, this predicate would be very large, with a quadratic number of rows for each set of equivalent expressions. By using a class as an intermediate step, the number of rows can be kept linear, and therefore can be cached.
 
-Example Queries
+ Example queries
```

Comment:
Sorry, on second thought, there's only one example... Perhaps change it to "Example query" or just "Example". Ditto for [line 98](https://github.com/Semmle/ql/pull/2160/files#diff-6368dc543c8e0bece775dbc865690c5eR98) of this file.

Reply:
I happy with both of those suggestions! Feel free to add them (if possible).

Reply:
I can't edit line 98, since it isn't in the diff :(